test/redirector-test.js

```diff
@@ -67,3 +67,17 @@ describe('Redirector Worker', function() {
     });
 });
 
+describe('experiment pages tests', function() {
+    it('should be able to check data from redirector', async function() {
+        expect(global.experimentPages).to.be.an('array');
+    });
+
+    // For discussion, it seems like global is only available inside a test, do we want to,
```

Comment:
Yeah workers are a little funky here as they run in an unusual context (neither browser nor node), so to make data available for testing, exposing context data on `global` is the easiest way. There are likely other ways we could test functionality, but those might involve deploying and creating a mock testing environment of sorts. That would be interesting to explore, but for the purpose of basic unit tests I think this is good enough?

Reply:
Sounds good, wanted to ask because it's a little bit hard to read the test results as it is, but we can refactor later for sure.